### ISSUE TITLE
MultiUploader can't use other FileInputType in gwt 2.7

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bdshadow/gwtupload#205]
```
What steps will reproduce the problem?
1. using gwt 2.7
2. new a MultiUploader with any FileInputType


What is the expected output? What do you see instead?
can't upload file ,don't have any message

What version of the product are you using? On what operating system?
GWT 2.7    smartgwt 5.0p     opensuse 13.1 with chrome 38

Please provide any additional information below.
thank's for your help

```

Original issue reported on code.google.com by `hgs1...@gmail.com` on 21 Nov 2014 at 10:57

**Comment by GoogleCodeExporter** [comment]
```
I have this issue also. For me, it does work with the BROWSER_INPUT 
FileInputType, but no others.

Using MultiUploader on GWT 2.7.

It doesn't give any messages or anything. The status just remains 
"UNINITIALIZED" even after selecting files.
```

Original comment by `raymond....@gmail.com` on 3 Dec 2014 at 11:08

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision ede6b80ed9fa.
```

Original comment by `manuel.carrasco.m` on 7 Jan 2015 at 3:22
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed not released.
Use 1.0.2-SNAPSHOT in the meanwhile I release 1.0.2

Thanks for reporting
```

Original comment by `manuel.carrasco.m` on 7 Jan 2015 at 3:28
- Added labels: **Milestone-1.0.2**
